src/extension/importer/core/src/main/java/org/geoserver/importer/FileData.java

```diff
@@ -28,7 +29,7 @@
     protected Resource file;
 
     public FileData(Resource file) {
-        this.file = file;
+        this.file = Resources.serializable(file);
```

Comment:
This is not quite what I expected, I expected something like:
````
protected String path;

protected Resource resource(){
     return resources.get( path );
}
````

But I would have to check how FileData is being used, perhaps this is what @cuttlefish was referencing in terms of accidentally loading a geotiff into jdbcstore.